modin/experimental/cloud/rpyc_proxy.py

```diff
@@ -173,6 +173,37 @@ def async_request(self, handler, *args, **kw):
                 return res
         return super().async_request(handler, *args, **kw)
 
+    def __patched_netref(self, id_pack):
```

Comment:
I am certainly ok with the changes, but wouldn't it be better to make a fork of `RPyC` into our organization and make changes to it? There is no specific code for modin...

Reply:
Just a thought for the future ...

Reply:
It complicates distributing things. What we should really do instead is to contribute fixes upstream.